hypex/dataset/dataset.py

```diff
@@ -252,9 +253,8 @@ def create_empty(backend=BackendsEnum.pandas, roles=None, index=None) -> "Datase
     def _convert_data_after_agg(self, result) -> Union["Dataset", float]:
         if isinstance(result, float):
             return result
-        return Dataset(
-            data=result, roles={column: StatisticRole() for column in self.roles}
-        )
+        role: ABCRole = StatisticRole()
```

Comment:
Didn't get wgy do we define "role" here and in the next line use it as an iterator.

Reply:
It is for mypy fixes, we need type initialization before usage